### ISSUE TITLE
--val step gives error if --val-workers* = 1

### ISSUE THREAD

**Issue by pieterlukasse** [opentargets/platform#522]
Steps to reproduce: 
1. run `--val` with `--val-workers-validator 1` and `--val-workers-writer 1`
2. this will result in error below:
```
Process Process-2:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/multiprocessing/process.py", line 267, in _bootstrap
    self.run()
  File "/usr/local/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python2.7/site-packages/pypeln/process.py", line 335, in _map
    _run_task(f_task, params)
  File "/usr/local/lib/python2.7/site-packages/pypeln/process.py", line 313, in _run_task
    params.stage_namespace.active_workers -= 1
  File "/usr/local/lib/python2.7/multiprocessing/managers.py", line 1023, in __getattr__
    return callmethod('__getattribute__', (key,))
  File "/usr/local/lib/python2.7/multiprocessing/managers.py", line 774, in _callmethod
    raise convert_to_error(kind, result)
RemoteError: 
---------------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/multiprocessing/managers.py", line 242, in serve_client
    obj, exposed, gettypeid = id_to_obj[ident]
KeyError: '7fb14f194650'
---------------------------------------------------------------------------
2019-03-28 12:24:25,237 mrtarget.modules.Evidences DEBUG - context teardown
2019-03-28 12:24:25,237 mrtarget.common.EvidencesHelpers DEBUG - flushing elasticsearch indexs
```

**Comment by afaulconbridge** [comment]
I can't recreate this with the current `master` - this might have been fixed by updating pypeln to 0.1.8 release recently.

Can we put this aside to be checked again with 19.04 ?

**Comment by pieterlukasse** [comment]
tried on 19.04.5 and seems to be solved now :+1:
